cmake/dependencies.cmake

```diff
@@ -1,8 +1,6 @@
 ## get project dependencies
 # Xlib
 find_package(X11 REQUIRED)
-# GLib (will be removed later)
-include(FindPkgConfig)
-pkg_check_modules(GLIB2 REQUIRED glib-2.0)
+#include(FindPkgConfig)
```

Comment:
Does this stay for any particular reason?

Reply:
all this cmake configuration is not clear to me, so I kept it because it might help at some point if we want to add new dependencies...

But actually you're right if you vote for dropping it.

Reply:
I've removed this line.